### ISSUE TITLE
[docker] nginx fails to start

### ISSUE THREAD

**Issue by TPGPL** [ppy/osu-web#4766]
```powershell
PS C:\users\mateu_000\documents\github\osu-web> docker-compose up
Starting osu-web_dependencies_1  ...
Starting osu-web_elasticsearch_1 ... Starting osu-web_notification-server_1 ...
doneting osu-web_db_1            ... done
Starting osu-web_php_1           ... done
Starting osu-web_dependencies_1        ... done
Starting osu-web_notification-server_1 ... done
Starting osu-web_nginx_1               ... error

ERROR: for osu-web_nginx_1  Cannot start service nginx: driver failed programming external connectivity on endpoint osu-
web_nginx_1 (0d86ddd26e83d98647c8685d827bed72780b5a7e7e582bd62962b669c8d54b35): Error starting userland proxy: /forwards
/expose/port returned unexpected status: 500

ERROR: for nginx  Cannot start service nginx: driver failed programming external connectivity on endpoint osu-web_nginx_
1 (0d86ddd26e83d98647c8685d827bed72780b5a7e7e582bd62962b669c8d54b35): Error starting userland proxy: /forwards/expose/po
rt returned unexpected status: 500
ERROR: Encountered errors while bringing up the project.```

**Comment by peppy** [comment]
Many solutions via google. Likely not an issue our end. Please google first.

Here's a promising one on checking for 22 seconds: https://github.com/docker/for-win/issues/573

**Comment by notbakaneko** [comment]
This is a docker issue.
restart docker

**Comment by TPGPL** [comment]
Solved by updating docker from 2.1 to the recent version 2.1.0.1
